src/Microsoft.AspNetCore.Server.Kestrel/KestrelServerLimits.cs

```diff
@@ -21,6 +24,30 @@ public class KestrelServerLimits
         private int _maxRequestHeaderCount = 100;
 
         /// <summary>
+        /// Gets or sets the maximum size of the response buffer before calls
+        /// write calls begin to block or return incomplete tasks.
+        /// </summary>
+        /// <remarks>
+        /// When set to null, the size of the response buffer is unlimited.
+        /// Defaults to 65536 bytes (64 KB).
+        /// </remarks>
+        public long? MaxResponseBufferSize
```

Comment:
nullable huh? With great power comes great responsibility. Netty and node both have a high and low watermark used for backpressure. Outgoing writes are halted when it hits the high watermark and continue once the low watermark is hit.